src/java.desktop/windows/native/libawt/windows/awt_PrintJob.cpp

```diff
@@ -528,184 +529,186 @@ Java_sun_awt_windows_WPageDialogPeer__1show(JNIEnv *env, jobject peer)
 
     setup.lStructSize = sizeof(setup);
 
-    HWND parentID = AwtPrintControl::getParentID(env, self);
-    if (parentID != NULL && ::IsWindow(parentID)) {
-        // windows native modality is requested (used by JavaFX).
-        setup.hwndOwner = parentID;
-    }
-    /*
-      Fix for 6488834.
-      To disable Win32 native parent modality we have to set
-      hwndOwner field to either NULL or some hidden window. For
-      parentless dialogs we use NULL to show them in the taskbar,
-      and for all other dialogs AwtToolkit's HWND is used.
-    */
-    else if (awtParent != NULL)
-    {
-        setup.hwndOwner = AwtToolkit::GetInstance().GetHWnd();
-    }
-    else
     {
-        setup.hwndOwner = NULL;
-    }
-
-    setup.hDevMode = NULL;
-    setup.hDevNames = NULL;
-    setup.Flags = PSD_RETURNDEFAULT | PSD_DEFAULTMINMARGINS;
-    // setup.ptPaperSize =
-    // setup.rtMinMargin =
-    // setup.rtMargin =
-    setup.hInstance = NULL;
-    setup.lCustData = (LPARAM)peerGlobalRef;
-    setup.lpfnPageSetupHook = reinterpret_cast<LPPAGESETUPHOOK>(pageDlgHook);
-    setup.lpfnPagePaintHook = NULL;
-    setup.lpPageSetupTemplateName = NULL;
-    setup.hPageSetupTemplate = NULL;
-
-
-    /* Because the return default flag is set, this first call
-     * will not display the dialog but will return default values, inc
-     * including hDevMode, hDevName, ptPaperSize, and rtMargin values.
-     * We can use the devmode to set the orientation of the page
-     * and the size of the page.
-     * The units used by the user is also needed.
-     */
-    if (AwtPrintControl::getPrintHDMode(env, self) == NULL ||
-        AwtPrintControl::getPrintHDName(env,self) == NULL) {
-        (void)::PageSetupDlg(&setup);
-        /* check if hDevMode and hDevNames are set.
-         * If both are null, then there is no default printer.
-         */
-        if ((setup.hDevMode == NULL) && (setup.hDevNames == NULL)) {
-            doIt = JNI_FALSE;
-            goto done;
+        HWND parentID = AwtPrintControl::getParentID(env, self);
```

Comment:
diff is making this really hard to compare line by line
So here you've just put the code in { .. } and indented ? 
What is the warning that motivates this ?
I don't see where you describe it.

Reply:
I haven't gotten around to converting this to the new approach of splitting declaration and assignment, but yes, that is all I've done. I've described the problem here above